src/scripts/build/uploadAutoCompleteLocations.js

```diff
@@ -27,6 +27,7 @@ import os from "os";
 import path from "path";
 import fs from "fs";
 import AdmZip from "adm-zip";
+import { uploadToS3 } from "../../utils/s3.js";
```

Comment:
Note the changed path - but also, @rhelmer, you explicitly made this into a dynamic import here:

https://github.com/mozilla/blurts-server/pull/4949/files#diff-86136336e578d7cf634398ffcbdfaf05b5b938e85866325dad589fb4ff98b191R331

Was that just to avoid importing it if not used? Because I think that potentially obscures the path being invalid, so I changed it to a regular import - but let me know if there's another reason.

Reply:
IIRC it was to be able to run this locally more easily when setting up a new dev env, I'm fine changing it as long as it works. It was added in https://github.com/mozilla/blurts-server/pull/4949